### ISSUE TITLE
Flurry custom Events not showing the page view counts in the dashboard

### ISSUE THREAD

**Issue by jesphinpt** [flurry/flurry-android-sdk#36]
I am using the flurry analytics. I have added custom events as well as per the [doc](https://developer.yahoo.com/flurry/docs/analytics/lexicon/pageviews/). The events are reporting as expected but it not showing the page views in the dashboard and it shows as 0. 

I have followed the doc for [iOS](https://developer.yahoo.com/flurry/docs/analytics/gettingstarted/events/ios/) and [Android](https://developer.yahoo.com/flurry/docs/analytics/gettingstarted/events/android/) and the events are captured with the Event Parameters.

```
`FlurryAgent.logEvent("User is on home screen");`
`FlurryAgent.logEvent("User logged in", user_detaills);`

```

Why we are not seeing the page views for our apps? Appreciate your input on this.

**Comment by poting-oath** [comment]
The Event analytics data is not Real-time, only the Sessions data is. You will need to wait for few hours to one day to see the Event data.

**Comment by jesphinpt** [comment]
@poting-oath I am seeing event in my dashboard. But the page views shows 0 or no data. Shall I need to add any other integration for viewing the page views for the app.
![page_view (1)](https://github.com/flurry/flurry-android-sdk/assets/67142639/793a23f5-46db-45ec-822d-364bf82124a9)

**Comment by jesphinpt** [comment]
@poting-oath Appreciate your inputs on this. 

Shall i need to add any additional code other that custom events.

**Comment by poting-oath** [comment]
@jesphinpt No, you don't need to do any other calls. Just the custom events.
BTW, what do you mean "the page views"? How do you navigate to this page?

**Comment by jesphinpt** [comment]
Login to flurry dashboard ==> Usage ==> App Activity ==> Scroll down

<img width="832" alt="Screenshot 2023-10-13 at 5 03 45 PM" src="https://github.com/flurry/flurry-android-sdk/assets/67142639/ff8ad163-58aa-4471-a8e9-0a7dbd6c65b2">

**Comment by poting-oath** [comment]
This feature has been removed from the Flurry SDK. I.e., `FlurryAgent.onPageView` is no longer available.
The Flurry UI keeps this page is only for backwards compatible.
